deployment/terraform/create.go

```diff
@@ -465,30 +467,35 @@ func (t *Terraform) createAdminUser(extAgent *ssh.ExtAgent) error {
 	return nil
 }
 
-func (t *Terraform) setDefaultTextSearchConfig(extAgent *ssh.ExtAgent, config string) error {
-	if t.config.TerraformDBSettings.InstanceEngine != "aurora-postgresql" {
-		return fmt.Errorf("default_text_search_config can only be set in postgres")
-	}
-
+func (t *Terraform) updatePostgresSettings(extAgent *ssh.ExtAgent) error {
 	dns := fmt.Sprintf("postgres://%s:%s@%s/%s?sslmode=disable",
 		t.config.TerraformDBSettings.UserName,
 		t.config.TerraformDBSettings.Password,
 		t.output.DBWriter(),
 		t.config.DBName(),
 	)
 
-	sqlCmd := fmt.Sprintf("ALTER DATABASE %s SET default_text_search_config TO \"%s\"",
+	sshc, err := extAgent.NewClient(t.output.Instances[0].PublicIP)
```

Comment:
I know this line was here before, but should we double-check that `t.ouput.Instances` has at least one instance and fail otherwise?